syntax = "proto3";
package netchasm;
message ThreadInfo {
  uint64 numThreads = 1;
  uint64 numIdleThreads = 2;
}
